pkgs/development/tools/build-managers/apache-maven/build-package.nix

```diff
@@ -44,6 +44,13 @@ stdenv.mkDerivation (rec {
     outputHashMode = "recursive";
     outputHash = mvnHash;
   } // mvnFetchExtraArgs);
+in
+stdenv.mkDerivation (builtins.removeAttrs args [ "mvnFetchExtraArgs" ] // {
+  inherit fetchedMavenDeps;
+
+  nativeBuildInputs = args.nativeBuildInputs or [ ] ++ [
```

Comment:
Can't believe this was missing.
probably should be a default argument to the function though so we can override it.